src/pages/cluster-management/clusterInfo.html

```diff
@@ -77,7 +77,7 @@ <h1>
                 ng-click="toggleLegend()"
                 tooltip="{{'cluster_management.cluster_page.toggle_legend_btn' | translate}}"
                 tooltip-placement="right">
-            <span class="icon-help legend-toggle-icon"></span>
+            <span class="icon-help toggle-legend--icon"></span>
```

Comment:
Is this double hyphen intended or it's hust a typo or you invented some sort of BEM css selector style here?!